config/versions.yaml

```diff
@@ -46,6 +46,13 @@ related-software:
       repository: "kabanero/kabanero-che"
       tag: "0.6.0"
 
+  codeready-workspaces:
+  - version: "0.6.0"
```

Comment:
I would prefer to see this version number match the version number of the operator.  The eclipse-che version was 0.6.0 long before Kabanero 0.6.0 existed, it just happened to catch up to the Kabanero number.  I don't think we intend to have a version of the devfile per kabanero release, right?  Maybe using the devfile release number would be more appropriate.

Reply:
Ed and I couldn't come up with a more appropriate identifier, so we're just going to leave it like this for now.